hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -142,6 +143,9 @@ public Response get(
         ozoneKeyIterator = bucket.listKeys(prefix, startAfter);
       } else if (startAfter == null && continueToken != null) {
         ozoneKeyIterator = bucket.listKeys(prefix, decodedToken.getLastKey());
+      } else if (OZONE_URI_DELIMITER.equals(delimiter)) {
```

Comment:
Will this delimiter parameter be specified together with startAfter and continueToken? I mean shall we handle delimiter in the ahead if or else if branch too?

Reply:
> Will this delimiter parameter be specified together with startAfter and continueToken? I mean shall we handle delimiter in the ahead if or else if branch too?

Make sense. Also looks OK in ahead case. I'll try to fix later.